.github/workflows/docker-ephemeral-env.yml

```diff
@@ -17,14 +17,12 @@ jobs:
         id: check
         shell: bash
         run: |
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Did you mean to delete the top two lines? I was assuming we just need to dedent the lower section.

Reply:
If you check other similar commands, this check is only to see if the variables are defined.